packages/typespec-azure-telemetry/src/utils/common-properties.browser.ts

```diff
@@ -0,0 +1,18 @@
+export function getCommonProperties() {
+  return {
+    sessionId: getRandomUUID(),
+    "telemetry.sdk.name": "typespec-azure-telemetry",
+    "telemetry.sdk.language": "javascript",
+  };
+}
+
+function getRandomUUID() {
+  try {
+    if (typeof crypto !== "undefined" && typeof crypto.randomUUID === "function") {
+      return crypto.randomUUID();
+    }
+  } catch {
+    // ignore error
+  }
+  return;
```

Comment:
what happens if it errors out? no session id or empty id?

Reply:
No sessionId in that case. It will still show up in app insights, but each event will be treated as being part of their own session.

That said - this file is mostly inconsequential for now since we'll use the NoOpTelemetryReporter in browser builds.